301-drupal8-vmss-glusterfs-mysql/azuredeploy.json

```diff
@@ -99,22 +99,23 @@
       "type": "string",
       "defaultValue": "no",
       "allowedValues": [
-        "no"
+        "no",
+        "yes"
       ],
       "metadata": {
         "description": "If Yes new MySQL server will be provisioned using the mysql replication template"
       }
     },
     
-    "mySqlFQDN": {
+    "existingMySqlFQDN": {
```

Comment:
Have changed the default configuration to create new MySQL server, and modified the default value of existingMySqlFQDN to "onlyRequiredForExistingMySQL"